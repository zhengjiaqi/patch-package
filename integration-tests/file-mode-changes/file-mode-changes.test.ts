import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({
  projectName: "file-mode-changes",
  shouldProduceSnapshots: true,
})
