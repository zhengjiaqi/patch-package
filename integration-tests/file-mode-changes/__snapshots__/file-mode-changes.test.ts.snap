// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test file-mode-changes: the patch file 1`] = `
"SNAPSHOT: the patch file
diff --git a/node_modules/prettier/bin-prettier.js b/node_modules/prettier/bin-prettier.js
old mode 100755
new mode 100644
diff --git a/node_modules/prettier/index.js b/node_modules/prettier/index.js
old mode 100644
new mode 100755
END SNAPSHOT"
`;
